speechbrain/lobes/models/HifiGAN.py

```diff
@@ -5,8 +5,9 @@
 For more details: https://arxiv.org/pdf/2010.05646.pdf
 
 Authors
- * Duret Jarod 2021
+ * Jarod Duret 2021
  * Yingzhi WANG 2022
+ * Jarod Duret 2023
 """
 
```

Comment:
I don't see any problem here but changes and adds must be validated by @BenoitWang

Reply:
Hi Yingzhi,
Yes, exactly and using reduced units allows faster training and inference times. 
I'd say that this is not required for TTS because, as you said, duration prediction is already incorporated into the text-to-mel model.